mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -3956,6 +3956,17 @@ struct OrderbookP2PItem {
     created_at: u64,
 }
 
+macro_rules! try_get_age_or_default {
+    ($created_at: expr) => {{
+        let now = now_sec();
+        now.checked_sub($created_at)
+            .ok_or_else(|| warn!("now - created_at: ({} - {}) caused a u64 underflow.", now, $created_at))
+            .unwrap_or_default()
```

Comment:
you can warn and return default in `unwrap_or_else`

Reply:
perhaps age should not be i64, i think it did not hurt if I changed a type of it. In documentation it stated as number.
Thank you, @ozkanonur for your sharp eye )
done

Reply:
You can stil avoid using `ok_or_else` and then `unwrap`.

Like:

```rs
now.checked_sub($created_at).unwrap_or_else(|| {
    warn..
    0 or Default::default()
});
```

Reply:
@rozhkovdmitrii should I merge this or will you implement the above suggestion?

Reply:
[done](https://github.com/KomodoPlatform/atomicDEX-API/pull/1851/commits/6b44667cbe396d193aa4cbb14b784a767702bb46)